sleap/gui/app.py

```diff
@@ -85,7 +86,8 @@
 from sleap.prefs import prefs
 from sleap.skeleton import Skeleton
 from sleap.util import parse_uri_path
-
+from sleap.gui.commands import CommandContext
+from sleap import session
```

Comment:
Also remove, not sure what this would import? We access `session` through `MainWindow.state["session"]`

```suggestion
```